i18n/pl.toml

```diff
@@ -0,0 +1,191 @@
+other = "Znany tez jako"
+
+[layouts_docs_glossary_description]
+other = "Celem tego słownika jest przedstawienie wszechstronnej, ujednoliconej listy terminologii związanej z Kubernetes. Słownik zawiera terminy specyficzne dla Kubernetes, a także pojęcia bardziej ogólne, dające lepsze zrozumienie kontekstu."
+
+[layouts_docs_glossary_deselect_all]
+other = "Odznacz wszystko"
+
+[layouts_docs_glossary_click_details_after]
+other = "po dokładniejsze wytłumaczenie."
+
+[layouts_docs_glossary_click_details_before]
+other = "Kliknij w"
+
+[layouts_docs_glossary_filter]
+other = "Znajdź pojęcia według etykiet"
+
+[layouts_docs_glossary_select_all]
+other = "Zaznacz wszystko"
+
+[layouts_docs_partials_feedback_improvement]
+other = "zaproponować poprawkę"
+
+[layouts_docs_partials_feedback_issue]
+other = "Otwórz zgłoszenie w repozytorium GitHub, jeśli chcesz "
+
+[layouts_docs_partials_feedback_or]
+other = "lub"
+
+[layouts_docs_partials_feedback_problem]
+other = "zgłosić problem"
+
+[layouts_docs_partials_feedback_thanks]
+other = "Dziękujemy za informację zwrotną. Jeśli masz konkretne pytanie dotyczące użycia Kubernetes, odwiedź"
+
+[layouts_docs_search_fetching]
+other = "Pobieram wyniki.."
+
+[main_by]
+other = "przez"
+
+[main_cncf_project]
+other = """Nasz projekt jest uznany przez <a href="https://cncf.io/">CNCF</a> za dojrzały</p>"""
+
+[main_community_explore]
+other = "Poznaj społeczność"
+
+[main_contribute]
+other = "Wnieś swój wkład"
+
+[main_copyright_notice]
+other = """The Linux Foundation &reg;. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">Trademark Usage page</a>"""
+
+[main_documentation_license]
+other = """The Kubernetes Authors | Documentation Distributed under <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC BY 4.0</a>"""
+
+[main_edit_this_page]
+other = "Edytuj stronę"
+
+[main_github_create_an_issue]
+other = "Zgłoś problem"
+
+[main_github_invite]
+other = "Chcesz zacząć współtworzyć kod Kubernetes?"
+
+[main_github_view_on]
+other = "Zajrzyj na GitHub"
+
+[main_kubernetes_features]
+other = "Funkcjonalności Kubernetes"
+
+[main_kubeweekly_baseline]
+other = "Zapisz się na KubeWeekly, jeśli jesteś zainteresowany najnowszymi wiadomościami o Kubernetes."
+
+[main_kubernetes_past_link]
+other = "Poprzednie newslettery"
+
+[main_kubeweekly_signup]
+other = "Zapisz się"
+
+[main_page_history]
+other ="Historia strony"
+
+[main_page_last_modified_on]
+other = "Ostatnia modyfikacja strony"
+
+[main_read_about]
+other = "Przeczytaj o"
+
+[main_read_more]
+other = "Przeczytaj więcej"
+
+[note]
+other = "Informacja:"
+
+[objectives_heading]
+other = "Cele"
+
+[prerequisites_heading]
+other = "Nim zaczniesz"
+
+[ui_search_placeholder]
+other = "Szukaj"
+
+[version_check_mustbe]
+other = "Twój serwer Kubernetes musi być w wersji "
```

Comment:
```suggestion
other = "Twój serwer Kubernetesa musi być w wersji "
```

Reply:
I'm not sure if this one should inflect.

Reply:
fine for me